src/pages/Browse/index.tsx

```diff
@@ -68,6 +96,7 @@ const mapStateToProps = (
   return {
     apiState: state.api,
     version: getVersionInfo(state.versions, parseInt(versionId, 10)),
+    file: getVersionFile('', state.versions, parseInt(versionId, 10)),
```

Comment:
I believe this is a code smell. We need to store the `path` we click on somehow, no?

Reply:
@bobsilverberg ^ (sorry I thought I mentioned you in the previous comment)

Reply:
I think this has now been dealt with, via PR #232, correct?

Reply:
yep :)